examples/Observe_example.ipynb

```diff
@@ -2,75 +2,55 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
    "id": "aba8b4d5-199f-4fe0-97a2-5a2a3f43cbf2",
    "metadata": {},
    "outputs": [],
    "source": [
     "import ipyreact\n",
-    "from traitlets import  Bool, Int, observe"
+    "from traitlets import Int, observe, Unicode"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": null,
    "id": "b11af66b-f256-4441-8c38-0283866024bd",
    "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "application/vnd.jupyter.widget-view+json": {
-       "model_id": "2ca30de3dae8466d925b92786125dc7e",
-       "version_major": 2,
-       "version_minor": 0
-      },
-      "text/plain": [
-       "OtherCountWidget()"
-      ]
-     },
-     "execution_count": 2,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
    "source": [
-    "class OtherCountWidget(ipyreact.ReactWidget):\n",
+    "def check_is_even(number):\n",
+    "    if number %2 == 0:\n",
+    "        return \"Yes ✅✅✅\" \n",
+    "    else:\n",
+    "        return \"No 🧊🧊🧊\"\n",
+    "\n",
+    "\n",
+    "class EvenOddWidget(ipyreact.ReactWidget):\n",
     "    #note that when we add these traitlets, they will automatically be made available to the react component\n",
-    "    is_even = Bool(False).tag(sync=True)\n",
+    "    message = Unicode(\"Click the button!\").tag(sync=True)\n",
     "    \n",
     "    count = Int(0).tag(sync=True)\n",
     "\n",
     "    #it is key that we also take the change argument, otherwise a hard to find python error will be thrown\n",
     "    @observe('count')\n",
     "    def _observe_count(self, change):\n",
-    "        new_val = self.count\n",
-    "        if new_val %2 == 0:\n",
-    "            self.is_even = True\n",
-    "        else:\n",
-    "            self.is_even = False\n",
-    "    \n",
+    "        self.message = check_is_even(self.count)\n",
```

Comment:
I think the `check_is_even` should not return a string, but a boolean, the widget should translate that to a string, sth like:
```suggestion
    "        self.message = "Yes ✅✅✅\" if check_is_even(self.count) else "No 🧊🧊🧊\"\n",
```